openstax/settings/base.py

```diff
@@ -216,3 +214,32 @@
 
 # WAGTAIL SETTINGS
 WAGTAIL_SITE_NAME = 'openstax'
+
+##################################
+#        ACCOUNTS SETTINGS       #
```

Comment:
On second thought.  I would like to keep the accounts settings here.  Mainly because since I'm adding the accounts app to the INSTALLED_APP list, I feel like there should be some default settings included that will allow the app to work properly on startup.